test-case-reporting/types/test-case-reporting.d.ts

```diff
@@ -119,4 +119,20 @@ declare module 'test-case-reporting' {
       time: number; // in milliseconds
     }
   }
+
+  namespace Travis {
+    export interface Report {
+      job: Travis.Job;
+      build: Travis.Build;
+      result: KarmaReporter.TestResultReport;
+    }
+    export interface Build {
+      buildNumber: string;
```

Comment:
Should this be type number?

Reply:
I made it a string because that's what Travis returns -- https://github.com/ampproject/amphtml/blob/master/build-system/common/travis.js#L52

I see the inconsistency, though -- would you recommend making the buildNumber a string across the board, a number across the board, something else?

Reply:
(I should definitely add a comment to the Travis namespace explaining it's supposed to match the Travis POST requests)

Reply:
You could cast it from string to number. It's just that it's misleading that `*Number` is a string. Up to you.

Reply:
I'll leave it as a string and add a clarifying comment